rr_viz/src/rr_viz_tabs.py

```diff
@@ -36,10 +41,9 @@ def __init__(self, parent):
         self.connection_label_text.setFont(QFont('Ubuntu', 14, QFont.Bold))
         self.connection_label_text.setAlignment(Qt.AlignRight | Qt.AlignVCenter)
         
-        self.connection_status = QLabel('Connection lost')
+        self.connection_status = QLabel('Lost')
```

Comment:
I would initialise this lablel as `Connecting` just to know if it has connected for the first time at some point